.changeset/plenty-apples-type.md

```diff
@@ -0,0 +1,8 @@
+---
+"@fuel-ts/account": patch
+"@fuel-ts/program": patch
+"@fuel-ts/script": patch
+"fuels": patch
```

Comment:
```suggestion
```

Just test changes so we don't need a redeploy, unless I've missed something.

Reply:
That's a good point, have we released patch updates for test changes in the past? cc @arboleya 

Otherwise the changeset is actually changed now due to the [removal of some files](https://github.com/FuelLabs/fuels-ts/pull/2811#issuecomment-2289135900)

17f9ad5

Reply:
No, unless it involves template stuff that needs to go with `create-fuels`.

Reply:
63e2c2e13